frontend/src/App.js

```diff
@@ -1,23 +1,48 @@
-import React from 'react'
+import React from "react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import Classes from './pages/classes';
-import VolunteerProfile from './pages/volunteerProfile';
-import VolunteerDash from './pages/volunteerDash';
+import "notyf/notyf.min.css";
+import Classes from "./pages/classes";
+import VolunteerProfile from "./pages/volunteerProfile";
+import VolunteerDash from "./pages/volunteerDash";
+import SignUpPage from "./pages/signup";
+import LoginPage from "./pages/login";
+import ForgotPasswordPage from "./pages/forgotPassword";
+import ResetPasswordPage from "./pages/resetPassword";
 
 function App() {
-
-  return (
-    <div className="App">
-      <BrowserRouter>
-        <Routes>
-          <Route path="/" element={<VolunteerDash/>} />
-          <Route path="/volunteer/classes" element={<Classes/>} />
-          <Route path="/volunteer/my-profile" element={<VolunteerProfile/>} />
-          <Route path="/volunteer/schedule" element={<VolunteerDash/>} />
-        </Routes>
-      </BrowserRouter>
-    </div>
-  );
+    return (
+        <div className="App">
```

Comment:
let's clean up the formatting here, keep it consistent, as it was before, to be one line of code per route.